pom.xml

```
@@ -58,7 +58,7 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>ssh-slaves</artifactId>
+            <artifactId>ssh-agents</artifactId>
```

That's an identifier, needs to remain the same
```suggestion
            <artifactId>ssh-slaves</artifactId>
```